portal-api/src/cron-scheduler-jobs/helpers/utilisation-report-helpers.test.js

```diff
@@ -291,10 +291,11 @@ describe('utilisation-report-helpers', () => {
       api.getAllBanks.mockResolvedValue([MOCK_BANKS.HSBC]);
 
       const existingReport = {
-        ...MOCK_UTILISATION_REPORT,
-        // 'month' should be 1-indexed, Date.getMonth() is 0-indexed, so this sets 'month' to the previous month
-        month: dueDate.getMonth(),
-        year: dueDate.getFullYear(),
+        ...aUtilisationReportResponse(),
+        reportPeriod: {
+          start: { month: dueDate.getMonth(), year: dueDate.getFullYear() },
```

Comment:
Ah whoops didn't notice I had deleted that comment! I've actually just rewritten this a little to help make it a little clearer what is going on. As a side note, it's best to avoid importing main code helper methods into tests when it can be avoided to keep the test as independent of mainline code as possible. So here I would advocate for not importing the helper here, and just manually setting valid months.